app/views/user_notifier_mailer/send_welcome_email.html.erb

```diff
@@ -0,0 +1,430 @@
+                            Educación, deportes, primera infancia, salud, alimentación y trabajo
+                            social.
+                          </span>
+                        </p>
+                      </div>
+
+                    </td>
+                  </tr>
+
+                  <tr>
+                    <td style="font-size:0px;word-break:break-word;">
+
+
+
+                      <div style="height:30px;">
+                        &nbsp;
+                      </div>
+
+
+
+                    </td>
+                  </tr>
+
+                  <tr>
+                    <td align="left" style="font-size:0px;padding:15px 15px 15px 15px;word-break:break-word;">
+
+                      <div
+                        style="font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:11px;line-height:1.5;text-align:left;color:#000000;">
+                        <p style="text-align: center;">Podes contactarnos por los siguientes medios:</p>
+                        <p style="text-align: center;">Mail: somosfundacionmas@gmail.com</p>
+                        <p style="text-align: center;">Instagram: SomosMás</p>
+                        <p style="text-align: center;">Facebook: Somos_Más</p>
+                        <p style="text-align: center;">Teléfono de contacto: 1160112988</p>
+                        
+                      </div>
+
+                    </td>
+                  </tr>
+
+                  <tr>
+                    <td style="font-size:0px;word-break:break-word;">
+
+
+
+                      <div style="height:30px;">
+                        &nbsp;
+                      </div>
+
+
+
+
+                    </td>
+                  </tr>
+
+                </table>
+
+              </div>
+
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+    </div>
+
+
+
+
+    <div style="Margin:0px auto;max-width:600px;">
+
+      <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation" style="width:100%;">
+        <tbody>
+          <tr>
+            <td style="direction:ltr;font-size:0px;padding:9px 0px 9px 0px;text-align:center;vertical-align:top;">
+
+
+              <div class="mj-column-per-100 outlook-group-fix"
+                style="font-size:13px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;">
+
+                <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;"
+                  width="100%">
+
+                  <tr>
+                    <td style="font-size:0px;word-break:break-word;">
+
+
+                      <div style="height:50px;">
+                        &nbsp;
+                      </div>
+
+
+
+                    </td>
+                  </tr>
+
+                </table>
+
+              </div>
+
+            </td>
+          </tr>
+        </tbody>
+      </table>
+
+    </div>
+
+
+
+  </div>
+
+</body>
+
+</html>
```

Comment:
If you are using VSCode, you can add an extra line automatically:
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files
```suggestion
</html>

```